cola/widgets/main.py

```diff
@@ -76,6 +76,10 @@ def __init__(self, model, parent=None, settings=None):
         # Runs asynchronous tasks
         self.runtask = qtutils.RunTask()
 
+        from cola.widgets.toolbar import create_toolbar
```

Comment:
Let's figure out why we have to defer this import.  If you pull it out to where the others are, what traceback do you see?